lib/livebook.ex

```diff
@@ -31,10 +31,15 @@ defmodule Livebook do
       config :livebook, LivebookWeb.Endpoint, http: [ip: ip]
     end
 
-    if password = Livebook.Config.password!("LIVEBOOK_PASSWORD") do
-      config :livebook, authentication_mode: :password, password: password
-    else
-      config :livebook, token: Livebook.Utils.random_id()
+    cond do
+      password = Livebook.Config.password!("LIVEBOOK_PASSWORD") ->
+        config :livebook, authentication_mode: :password, password: password
+
+      Livebook.Config.token_enabled!("LIVEBOOK_TOKEN_ENABLED") ->
+        config :livebook, token: Livebook.Utils.random_id()
```

Comment:
```suggestion
        config :livebook, authentication_mode: :token, token: Livebook.Utils.random_id()
```

For clarity?

Reply:
Haha, that's what I did, but then I reliased we use different configuration for tests:

https://github.com/livebook-dev/livebook/blob/79049e7f8cb5a775bb258ca112174cf2c5e8ae00/config/test.exs#L12-L13

And this would override it. WDYT?

Reply:
Hrm, this means no one can change the authentication mode with a regular config. Maybe we should flip this around, instead of disabling token, we should have `LIVEBOOK_DISABLE_AUTH` and then we only change the auth stuff if the LIVEBOOK_PASSWORD or LIVEBOOK_DISABLE_AUTH are set.

Reply:
The point is that with the current config they can change it, because we only set the token and leave `:authentication_mode` as is. And I think it's fine that we always override the token, because it's exactly the point that it's random and generated, no?

Reply:
Maybe it would be more clear if we always configure `:token`, regardless of the auth mode?

Reply:
Nah, ignore me. This is good to go then!